recipe/meta.yaml

```diff
@@ -1,21 +1,16 @@
-{% set version="3.2.3.1" %}
+{% set version="3.3.0" %}
 
 package:
     name: pytables
     version: {{ version }}
 
 source:
-    fn: PyTables-v.{{ version }}.tar.gz
-    url: https://github.com/PyTables/PyTables/archive/v.{{ version }}.tar.gz
-    sha256: e422a57f0507d03c4bdbf1629b6abcd1faa83338cd2fce4747e8390d3290c556
-    patches:
-        - setup.patch
-        - win-c.patch  # [win]
-        - tables_init.patch  # [win]
```

Comment:
@ocefpaf the patches where mostly needed for the windows builds. They were incorporated upstream, but unreleased until 3.3.0 (today). 

The patches are no longer needed.